oaknational/Oak-Web-Application
# feat: add posts to homepage

## Description
- Add blog posts to homepage (and webinars, but commented out for now)
- Some other work on bringing webinar pages in line with more recent blog changes

Extra styling work TBD in a follow-up PR

## Issue(s)

Progress on #508 

## How to test

Go to https://oak-web-application-git-feat-home-blogs-oak-national-academy.vercel.app/
You should see a list of blog posts
- Clicking on a blog post should take you to the post detail page
- Clicking "All blogs" should take you to the blog listing page

## Screenshots
![Google Chrome - 2022-08-26 at 12 55 47](https://user-images.githubusercontent.com/2717635/186898188-7dcc6aca-dd40-4f9c-bb5a-a9bd05234091.png)
![Google Chrome - 2022-08-26 at 12 56 35](https://user-images.githubusercontent.com/2717635/186898263-2584a1c3-f29a-43db-8971-9fad95ad4678.png)


## Checklist
